symfony/flex/1.0/.env

```diff
@@ -9,6 +9,7 @@
 # Real environment variables win over .env files.
 #
 # DO NOT DEFINE PRODUCTION SECRETS IN THIS FILE NOR IN ANY OTHER COMMITTED FILES.
+# https://symfony.com/doc/current/best_practices.html#use-secret-for-sensitive-information
```

Comment:
What about linking here directly?

https://symfony.com/doc/current/configuration/secrets.html

Reply:
hi @Nyholm 
I wanted to point to the same page as the like below
But we can define the link that is more relevant for you :)

Reply:
friendly ping @Nyholm

Reply:
I’ll do it and swap and finish this PR
Thx for feed-back

Reply:
done in https://github.com/symfony/recipes/pull/1074
closing here